aiml_utils/logging_utils.py

```diff
@@ -0,0 +1,16 @@
+__author__ = 'chloe'
+
+import logging
+import sys
+
+
+class LoggingUtils:
+
+    log = logging.getLogger(__name__)
+
+    def __init__(self):
+        out_hdlr = logging.StreamHandler(sys.stdout)
+        out_hdlr.setFormatter(logging.Formatter('%(asctime)s %(message)s'))
+        out_hdlr.setLevel(logging.INFO)
```

Comment:
Setting the level twice. Could be a constant, or set as a var once and used in the two setLevels (Why are there two?)